openedx/features/course_experience/api/v1/views.py

```diff
@@ -30,3 +36,44 @@ def reset_course_deadlines(request):
         return Response({'message': 'Deadlines successfully reset.'})
     except Exception:
         raise UnableToResetDeadlines
+
+
+class CourseDeadlinesMobileView(RetrieveAPIView):
```

Comment:
Unsure if this is the best naming scheme.

Reply:
I am not good at names (shrug)